src/libraries/Microsoft.Extensions.Options/src/OptionsWrapper.cs

```diff
@@ -1,6 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
+using System.ComponentModel;
 using System.Diagnostics.CodeAnalysis;
```

Comment:
why adding these here? wasn't the code compiling before adding them?

Reply:
These were here from when I previously added obsolete API.  Missed cleaning these up.  Thanks!